.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: go get && go build ./... && go test ./... && make
+    command: go run
+
+
```

Comment:
Usually local development tools files are not merged in the repo.
It is just a matter of adding `.gitpod.yml` to the [.gitignore file](https://github.com/vmware-tanzu/kubeapps/blob/main/.gitignore).